code/game/gamemodes/wizard/artifacts.dm

```diff
@@ -58,6 +58,12 @@
 	/obj/item/clothing/suit/space/rig/wizard,\
 	/obj/item/clothing/head/helmet/space/rig/wizard)
 
+/datum/spellbook_artifact/scrying/purchased(mob/living/carbon/human/H)
```

Comment:
I just noticed that this PR isn't giving wizards who buy the hardsuit no breath, its giving those who buy the scrying orb no breath

Reply:
Okay, this is epic